dojo/jira_link/helper.py

```diff
@@ -77,7 +77,7 @@ def get_jira_project(obj, use_inheritance=True):
 
     if isinstance(obj, Finding):
         finding = obj
-        return get_jira_project(finding.test)
+        return get_jira_project(finding.test.engagement)
```

Comment:
my idea was to have the cascading of jira_project up through the hiearchie in one place only so I decided to go from finding to test and not straight to engagement. if we ever get jira config per test, we again will have to modify multiple places.

Reply:
I can understand why this was done this way with the possible intro of jira for tests, but otherwise its just multiple calls to the same function where one could do the trick. But we can keep it the way it is I suppose.